docs/user-guide/hardware-recommendations.md

```diff
@@ -0,0 +1,17 @@
+## Hardware recommendations
+
+As cardano-wallet runs side by side with cardano-node, the hardware requirements for cardano-wallet would largely depend on the hardware requirements for cardano-node. Current hardware requirements for cardano-node are published on cardano-node's [release page](https://github.com/input-output-hk/cardano-node/releases). For most cases cardano-node's hardware requirements are good enough to cover requirements of running cardano-wallet as well.
+
+Here are some general hardware recommendations for running cardano-wallet:
+
+**Processor:** A multicore processor with a clock speed of at least 1.6 GHz or faster is recommended, but a faster processor is better for optimal performance.
+**Memory:** A minimum of 4 GB of RAM is recommended, although more is better. 
+**Storage:** A minimum of 5 GB of free storage for wallet data is recommended.
+**Network:** A stable internet connection with at least 5 Mbps upload and download speeds is recommended for syncing the blockchain data and performing transactions.
```

Comment:
```suggestion
* **Processor:** A multicore processor with a clock speed of at least 1.6 GHz or faster is recommended, but a faster processor is better for optimal performance.
* **Memory:** A minimum of 4 GB of RAM is recommended, although more is better. 
* **Storage:** A minimum of 5 GB of free storage for wallet data is recommended.
* **Network:** A stable internet connection with at least 5 Mbps upload and download speeds is recommended for syncing the blockchain data and performing transactions.
```

I'm not entirely certain how the markdown will be rendered, but in order to put these items on separate lines, I think we need to do something, e.g. use unordered list formatting.